code/modules/custom_ka/cells.dm

```diff
@@ -7,7 +7,7 @@
 		if(stored_charge >= cell_increase)
 			to_chat(user, SPAN_WARNING("The pump on \the [src] refuses to move."))
 		else
-			if(!pump_delay || do_after(user, pump_delay))
+			if(!pump_delay || do_after(user, pump_delay, null, DO_SHOW_PROGRESS|DO_USER_CAN_MOVE|DO_USER_CAN_TURN))
```

Comment:
```suggestion
			if(!pump_delay || do_after(user, pump_delay, do_flags = (DO_SHOW_PROGRESS|DO_USER_CAN_MOVE|DO_USER_CAN_TURN)))
```
Or equivalent, basically avoid a null parameter and use the named one for the flags